docs/source/kiva/compiled_path_ex.py

```diff
@@ -17,4 +17,4 @@
     gc.add_path(path)
     gc.fill_path()
 
-gc.save("example.png")
+gc.save("compiled_path_ex.png")
```

Comment:
the changes to this file seem unrelated to the main purpose of this PR, but even so the changes make sense as something we want to do

Reply:
You're right, it's not related. I had made a similar change in the new state example (which was copied from the compiled path example)